org.opent2t.sample.thermostat.superpopular/com.nest.thermostat/js/thingTranslator.js

```diff
@@ -15,12 +15,63 @@ function validateArgumentType(arg, argName, expectedType) {
     }
 }
 
+function deviceHvacModeToTranslatorHvacMode(mode) {
+    switch (mode) {
+        case 'cool':
+            return 'coolOnly';
+        case 'heat':
+            return 'heatOnly';
+        case 'heat-cool':
+            return 'auto';
+        case 'off':
+            return 'off';
+    }
+
+    return undefined;
+}
+
+function translatorHvacModeToDeviceHvacMode(mode) {
+    switch (mode) {
+        case 'coolOnly':
+            return 'cool';
+        case 'heatOnly':
+            return 'heat';
+        case 'auto':
+            return 'heat-cool';
+        case 'off':
+            return 'off';
+    }
+
+    return undefined;
+}
+
```

Comment:
i feel like a dictionary look up may be a better pattern. dictionary which contains the data, functions are small and reusable. not a blocker